llvmlite/binding/targets.py

```diff
@@ -6,10 +6,28 @@
                     c_void_p, string_at, byref)
 
 from . import ffi
-from .module import parse_assembly
 from .common import _decode_string, _encode_string
 
 
+def get_process_triple():
+    """
+    Returns the target triple of the process when using jit for the current
+    process.
+    """
+    with ffi.OutputString() as out:
+        ffi.lib.LLVMPY_GetProcessTriple(out)
+        return str(out)
+
+def get_host_cpu_features():
+    """
+    Returns a string (e.g. "+sse,-avx) indicating the available CPU features.
+    May return an empty string if LLVM cannot detect the CPU features.
+    """
+    with ffi.OutputString() as out:
+        ffi.lib.LLVMPY_GetHostCPUFeatures(out)
+        return str(out)
```

Comment:
Ah, I had forgotten about create_target_machine... The thing is, if you want to change stuff in that string, it's not very pretty: you have to resort to string or regex matching.
You mean a string that would also act as a dict? That's ugly.